base_tier_validation/models/res_users.py

```diff
@@ -24,7 +23,7 @@ def review_user_count(self):
             if reviews:
                 records = (
                     self.env[model]
-                    .with_user(self.env.user)
+                    .sudo()
```

Comment:
and this

Reply:
Without this. There was an error in the tests. The user is not allowed to acces SO records.

Reply:
@bosd I think it is better to use `sudo` or `with_user` in the test. Did you try using these options instead of changing the code to make the test work?

Reply:
It was a long time ago I was working on this. I think I just changed it to make the test work.

Reply:
Test pass as well now by using `with_user`. So reverted this change.